code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm

```diff
@@ -0,0 +1,366 @@
+GLOBAL_LIST_EMPTY(zombies)
+
+/mob/living/simple_animal/hostile/abnormality/thunder_bird
+	name = "Thunderbird Altar"
+	desc = "An ominous totem built from the corpses of unusual creatures, crowned with the visage of its namesake in wood."
+	icon = 'ModularTegustation/Teguicons/48x64.dmi'
+	icon_state = "thunderbird"
+	icon_living = "thunderbird"
+	icon_dead = "thunderbird_dead"
+	speak_emote = list("intones")
+	var/list/thunder_bird_lines = list(
+				"Prostrate yourself! Harder!",
+				"Do you think I am happy, feather? Think again!",
+				"You folk, nothing but sacrifices. Sacrifices for me and everyone else here!",
+				"Your kind can never be forgiven!",
+				"Look around, you monsters! You've destroyed my people and nature!"
+				)
+	//Ideally it should only glow in its breached state
+	light_color = LIGHT_COLOR_BLUE
+	light_range = 5
+	light_power = 7
+
+	pixel_x = -8
+	base_pixel_x = -8
+
+	//suppression info
+	maxHealth = 2500
+	health = 2500
+	speed = 2
+	move_to_delay = 3
```

Comment:
Increase to 4. It has area denial and a dash that knocks down, it doesn't need to be this fast